### ISSUE TITLE
Code highlighted in single line

### ISSUE THREAD

**Issue by fsimkovic** [demoneaux/reveal-code-focus#9]
I'm trying to get the following slide to display properly but something is going terribly wrong.
```html
<section data-markdown data-menu-title="Example 1: Conversion">
    <script type="text/template">
         ### Example 1: Conversion
                        
         ```bash
         $> conkit.convert example.mat ccmpred example.rr casprr
         ```
                        
         ```python
         >>> import conkit
         >>> conkit.io.convert('example.mat', 'ccmpred', 'example.rr', 'casprr')
         ```
    </script>
</section>
```
This does provide the correct color coding, but ignores line breaks. 

![img01](https://cloud.githubusercontent.com/assets/6966675/21401103/ecddded8-c7aa-11e6-9beb-8152c1fc2a28.png)

Wrapping the code into ``<pre><code> ... </code></pre>`` tags yields the same result.

---

I'm using ``highlight.js`` v9.8.0 and the ``Tomorrow Night Eighties`` theme imported in the ``<head>`` of my ``index.html``.
```html
<link rel="stylesheet" href="bower_components/highlightjs/styles/tomorrow-night-eighties.css">
```

**Comment by demoneaux** [comment]
Thanks for the report! I'm currently on holiday, and I'll look at it on Monday! Cheers and have a great weekend!

**Comment by demoneaux** [comment]
Taking a brief look at this, this issue is because reveal-code-focus doesn't support multiple code blocks in 1 slide yet. For now you can try separating the code blocks into different slides, I'll try to come up with a fix after the weekend.

**Comment by fsimkovic** [comment]
Thanks @demoneaux for looking into this so quickly. Unfortunately, I will need to have the blocks on the same slide to compare them. Is there a possible "easy" fix of separating them via a keyword, i.e. use a custom html tag between the blocks?!

**Comment by demoneaux** [comment]
Actually, this doesn't appear to be a bug. In order for us to focus on each line, we modify the highlighted element, to wrap each line in a `<span class="line">`, while removing the `<br>` tags in the element. The span element allows the whole line to be selected and styled one at a time, but since the `<br>` tags are removed, they will appear in 1 single line, as span elements are `display: inline`. The recommended fix, which is in the usage demo, is to add `.line { display: block; }` to your CSS. I’ll try to make that clearer in the documentation.

**Comment by fsimkovic** [comment]
Brilliant, I completely missed that but indeed it resolved the issue. Thanks a lot for the help.

**Comment by demoneaux** [comment]
Closed in 7eddf8cf, where I shifted the styles slide under the main usage section, and added a description of the important `.line { display: block }`. Also, I'll be tracking support for multiple code blocks in #10.
